### DISCUSSION TITLE
From datm to create_shr_strdata_nml

### DISCUSSION THREAD

**Discussion by changliao1025** [changliao1025/liao-etal_2022_h2sc_gmd#19] [General]
<img width="593" alt="image" src="https://user-images.githubusercontent.com/20618384/222590252-9ecc4235-b4ce-4b69-aece-021ebab05eb1.png">

**Comment by changliao1025** [comment]
<img width="559" alt="image" src="https://user-images.githubusercontent.com/20618384/222594155-3cceacfa-c662-4328-925a-a6367e443adf.png">

**Reply by changliao1025** [reply]
@donghuix Do you have any idea how the datamode is set in the XML?

**Reply by xdongh** [reply]
> @donghuix Do you have any idea how the datamode is set in the XML?

For DATM, use ./xmlchange DATM_MODE = xxx
